packages/databases-collections/src/components/collection-fields/fle2-fields.jsx

```diff
@@ -62,6 +62,14 @@ function FLE2Fields({
   fle2,
   configuredKMSProviders,
 }) {
+  const [keyEncryptionKeyEditorText, setKeyEncryptionKeyEditorText] = useState(
+    () => {
+      return (
+        fle2.keyEncryptionKey || keyEncryptionKeyTemplate[fle2.kmsProvider]
+      );
+    }
+  );
```

Comment:
Out of curiosity, why are we using a function for setting this default value? 
```suggestion
  const [keyEncryptionKeyEditorText, setKeyEncryptionKeyEditorText] = useState(
    fle2.keyEncryptionKey || keyEncryptionKeyTemplate[fle2.kmsProvider]
  );
```
Would that also work?

Reply:
Just a habit, not really necessary here and probably reads a bit easier without a function, I'll replace, thanks for highlighting